doc/decisions/example_migrations/normalized-plants-and-ranks/2023-04-07-130215_plant_relationships/up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TYPE taxonomic_rank AS ENUM ('family', 'subfamily', 'genus', 'species', 'variety');
+ALTER TABLE plants
+    ADD COLUMN rank TAXONOMIC_RANK NOT NULL,
+    ADD COLUMN family_id INTEGER REFERENCES plants (id) NULL,
+    ADD COLUMN subfamily_id INTEGER REFERENCES plants (id) NULL,
+    ADD COLUMN genus_id INTEGER REFERENCES plants (id) NULL,
+    ADD COLUMN species_id INTEGER REFERENCES plants (id) NULL;
+
+
+CREATE TYPE relationship_kind AS ENUM ('companion', 'antagonist');
+CREATE TABLE plant_relationships (
+    id SERIAL PRIMARY KEY NOT NULL,
+    strength INTEGER CHECK (strength >= 0) NOT NULL,
```

Comment:
confidence and not strength

Reply:
See https://github.com/ElektraInitiative/PermaplanT/pull/240/commits/6db0bf265d61867ca68d660f2079dc152d6ba4e7